src/Tasks/Microsoft.NET.Build.Tasks/CreateComHost.cs

```diff
@@ -20,19 +20,85 @@ public class CreateComHost : TaskWithAssemblyResolveHooks
         [Required]
         public string ClsidMapPath { get; set; }
 
+        public ITaskItem[] TypeLibraries { get; set; }
+
         protected override void ExecuteCore()
         {
             try
             {
+                if (!TryCreateTypeLibraryIdDictionary(TypeLibraries, out var typeLibIdMap))
```

Comment:
Where do we want to validate the paths contained in those items exist? Do we defer that to the `ComHost` logic?

Reply:
Yes, we check the path inside the `ComHost`.